magefile.go

```diff
@@ -338,11 +337,18 @@ func PythonEnv() error {
 	return err
 }
 
-func BuildOpaBundle() error {
+func BuildOpaBundle() (err error) {
 	owner := "elastic"
 	r := "csp-security-policies"
 	cspPoliciesPkgDir := "/tmp/" + r
 
+	defer func() {
```

Comment:
I think better to just use the `tempDir` 
https://go.dev/src/io/ioutil/tempfile.go